From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: fix powerpc detection for recent linux kernels

We get this for recent kernels:

powerpc-603e-linux-gnu-gcc  -O2 -g  -DFEAT_RTC=1 -DLINUX -DHAS_STDINT_H -DHAS_INTTYPES_H -c rtc_linux.c
In file included from rtc_linux.c:61:
io_linux.h:39:2: error: #error "I don't know the values of the _IOC_* constants for your architecture"

because __ppc__ and __ppc64__ has changed into __powerpc__.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 io_linux.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: chrony-1.23/io_linux.h
===================================================================
--- chrony-1.23.orig/io_linux.h
+++ chrony-1.23/io_linux.h
@@ -16,7 +16,7 @@
 #define CHRONY_IOC_WRITE	1U
 #define CHRONY_IOC_READ	2U
 
-#elif defined(__alpha__) || defined(__sparc__) || defined(__ppc__) || defined(__ppc64__) || defined(__sparc64__)
+#elif defined(__alpha__) || defined(__sparc__) || defined(__ppc__) || defined(__ppc64__) || defined(__powerpc__) || defined(__sparc64__)
 #define CHRONY_IOC_NRBITS	8
 #define CHRONY_IOC_TYPEBITS	8
 #define CHRONY_IOC_SIZEBITS	13
